R/splitOn.R

```diff
@@ -90,49 +92,49 @@ setGeneric("splitOn",
             function(x, ...)
                 standardGeneric("splitOn"))
 
-# This function collects f (grouping variable), by, and 
+# This function collects group (grouping variable), by, and 
 # use.names and returns them as a list.
 .norm_args_for_split_by <- function(
-        x, f, by = MARGIN, MARGIN = NULL, use.names = use_names,
+        x, group = f, f, by = MARGIN, MARGIN = NULL, use.names = use_names,
         use_names = TRUE, ...){
     # input check
-    # Check f
-    if(is.null(f)){
-        stop("'f' must either be a single non-empty character value or",
+    # Check group
+    if(is.null(group)){
+        stop("'group' must either be a single non-empty character value or",
             " vector coercible to factor alongside the one of the dimensions of 'x'",
             call. = FALSE)
     }
     # Check by
     if( !is.null(by) ){
         by <- .check_MARGIN(by)
     }
-    # If f is a vector containing levels
-    if( !.is_non_empty_string(f) ){
+    # If group is a vector containing levels
+    if( !.is_non_empty_string(group) ){
         # Convert into factors
-        f <- factor(f, unique(f))
-        # Check if the length of f matches with one of the dimensions
-        if(!length(f) %in% dim(x)){
-            stop("'f' must either be a single non-empty character value or",
+        group <- factor(group, unique(group))
+        # Check if the length of group matches with one of the dimensions
+        if(!length(group) %in% dim(x)){
+            stop("'group' must either be a single non-empty character value or",
```

Comment:
revert changes in internal functions. splitOn already catches f/group parameter. (Keep only those messages that you have changed)